code/game/machinery/computer/buildandrepair.dm

```diff
@@ -111,3 +111,4 @@
 				var/B = new src.circuit.build_path ( src.loc )
 				src.circuit.construct(B)
 				qdel(src)
+	return TRUE
```

Comment:
there's a few nested returns above under istypes that may want to be return TRUE